packages/messenger-widget/src/components/RightHeader/NormalView.tsx

```diff
@@ -31,6 +31,7 @@ export function NormalView() {
             await getAvatarProfilePic(
                 mainnetProvider,
                 account?.ensName as string,
+                dm3Configuration.addressEnsSubdomain,
```

Comment:
Instead of passing the address via function args use the Cpntext directly from with the AvatarProfilePic component. Do this for all components.